creator-node/src/snapbackSM/syncHistoryAggregator.js

```diff
@@ -99,11 +99,11 @@ class SyncHistoryAggregator {
   /**
    * Returns the aggregate sync data of the current day's number of successful, failed,
    * and triggered syncs
-   * @param {Object} logContext the log context off of the Express req object
+   * @param {Object?} logContext the log context off of the Express req object
    * @returns an object of the current day's aggregate sync count like
    *    {triggered: <natural number>, success: <natural number>, fail: <natural number>}
    */
-  static async getAggregateSyncData (logContext) {
+  static async getAggregateSyncData (logContext = {}) {
```

Comment:
yeah, in this case, I can pass in the logContext from the verbose health check route. also adding a default log context with the required name param

Reply:
actually, just checking on the docs once more, I think it's perfectly fine to pass in an empty obj. when creating a child logger, it just appends extra fields to the logger 
https://github.com/trentm/node-bunyan/blob/0ff1ae29cc9e028c6c11cd6b60e3b90217b66a10/README.md#logchild

so bc there's nothing for the default, it'll use the parent configs in the logging.js file

im going to leave this as is as I dont think there's a point to adding more info to the logger since this is so far only called in the health check route

Reply:
awesome, thanks for confirming!